calculator/exceptions.py

```diff
@@ -1,7 +1,14 @@
 
 class InvalidOperation(Exception):
-    pass
+    def __init__(self, operation):
+        self.operation = operation
+        if (self.operation == 1):
+            Exception.__init__(self, 'Given operation is invalid.')
+        else:
+            Exception.__init__(self, '"{}" operation not supported.'.
+                           format(self.operation))
 
 
 class InvalidParams(Exception):
-    pass
+    def __init__(self):
+        Exception.__init__(self, 'Given params are invalid.')
```

Comment:
So from what I gather, super() allows you to the call base class with having to explicitly define it. IE in this case

`super(InvalidParams,self).__init__('Given params are invalid'`

Is that best practice? Is this benefit just that if the Base class changed we wouldn't have to refactor the explicit calls to it within the class? Is it wrong to do it the way we did in such a simple class or should we just always prefer to use super()?